server/src/controllers/ChapterUser/resolver.ts

```diff
@@ -140,6 +156,13 @@ async function updateInstanceRoleForChapterRoleChange({
       where: { id: user.id },
     });
   }
+  if (email && emailSubject && emailBody) {
+    await new MailerService({
+      emailList: email,
+      subject: emailSubject,
+      htmlEmail: emailBody,
+    }).sendEmail();
+  }
```

Comment:
This mixes concerns and would be better as two functions.  Previously `updateInstanceRoleForChapterRoleChange` did exactly that, but these additions make that name a lie.

Instead it's perfectly fine to create a helper function `emailUserAboutRoleChange`, say, and call that with whatever it needs (user, role, etc)